Test/Unit/Helper/CartTest.php

```diff
@@ -3739,7 +3739,7 @@ public function collectDiscounts_withCouponCode_collectsCouponCodeDiscount()
         $expectedTotalAmount = $totalAmount - $expectedDiscountAmount - $expectedDiscountAmountNoCoupon;
         $expectedDiscount = [
             [
-                'description' => trim(__('Discount ') . self::COUPON_DESCRIPTION),
+                'description' => trim(__('Discount ') . self::COUPON_DESCRIPTION . ' (' . self::COUPON_CODE . ')'),
```

Comment:
seems like using curly braces might be a bit more succinct? same in the two places below

Reply:
though you have to double quotes

Reply:
maybe I just did it wrong, but that didn't like the `self` variables.

Reply:
well i'm certainly not a php expert so if it doesn't work it doesn't work.